src/plugins/console/public/application/containers/editor/legacy/console_editor/editor_output.tsx

```diff
@@ -86,15 +88,19 @@ function EditorOutputUI() {
         data
           .map((result) => {
             const { value, contentType } = result.response;
+
+            let editorOutput;
             if (readOnlySettings.tripleQuotes && isJSONContentType(contentType)) {
-              return safeExpandLiteralStrings(value as string);
+              editorOutput = safeExpandLiteralStrings(value as string);
             }
+
             if (isMapboxVectorTile(contentType)) {
-              return i18n.translate('console.outputCannotPreviewBinaryData', {
-                defaultMessage: 'Cannot preview binary data.',
-              });
+              const vectorTile = new VectorTile(new Protobuf(value as ArrayBuffer));
+              const vectorTileJson = convertMapboxVectorTileToJson(vectorTile);
+              editorOutput = safeExpandLiteralStrings(vectorTileJson as string);
             }
-            return value;
+
+            return editorOutput;
```

Comment:
If the content is neither JSON nor MapboxVectorTIel, the variable `editorOuput` will remain `undefined`. Is this intentional? If not, we could either initialize `editorOuput` with `value` or return `value` as it was done previously.

Reply:
Thanks, valid point. Fixed.